draft-ietf-rtcweb-jsep.xml

```diff
@@ -561,7 +561,9 @@ setRemote(ANSWER) |                                   |
             <t>The candidate details are specified in an IceCandidate
             field, using the same SDP syntax as the
             "candidate-attribute" field defined in
-            <xref target="RFC5245" />, Section 15.1. For example:</t>
+            <xref target="RFC5245" />, Section 15.1. Note that this field
+            does not contain an "a=" prefix, as indicated in
+            the following example:</t>
```

Comment:
Nit: "As indicated in the following example, this field does not contain an "a=" prefix"